account.go

```diff
@@ -0,0 +1,27 @@
+package crunchyroll
+
+// Account contains information about a crunchyroll account.
+type Account struct {
+	AccountID     string `json:"account_id"`
+	ExternalID    string `json:"external_id"`
+	EmailVerified bool   `json:"email_verified"`
+	Created       string `json:"created"`
```

Comment:
The account created value is stored as RFC3339 value, thus `time.Time` instead of `string` can be safely be used here